man/docker-logs.1.md

```diff
@@ -42,11 +42,17 @@ logging drivers.
 **--tail**="all"
    Output the specified number of lines at the end of logs (defaults to all logs)
 
-The `--since` option shows only the container logs generated after
-a given date. You can specify the date as an RFC 3339 date, a UNIX
-timestamp, or a Go duration string (e.g. `1m30s`, `3h`). Docker computes
-the date relative to the client machine’s time. You can combine
-the `--since` option with either or both of the `--follow` or `--tail` options.
+The `--since` option can be Unix timestamps, date formated timestamps, or Go 
+duration strings (e.g. `10m`, `1h30m`) computed relative to the client machine’s 
+time. Supported formats for date formated time stamps include RFC3339Nano, 
+RFC3339, `2006-01-02T15:04:05`, `2006-01-02T15:04:05.999999999`, 
+`2006-01-02Z07:00`, and `2006-01-02`. The local timezone on the client will be 
+used if you do not provide either a `Z` or a `+-00:00` timezone offset at the 
+end of the timestamp.  When providing Unix timestamps enter 
+seconds[.nanoseconds], where seconds is the number of seconds that have elapsed 
```

Comment:
Or fractional second... but let's consider...  

The shift is hidden by the code, the precision is to nanoseconds because of the unix time api limitations.  The traditional unix timestamp says it's nanoseconds. But you are right in that because it's not a separate value it's not really nanoseconds it's subseconds up to or exceeding the precision of nanoseconds.  But alas that gets a bit long winded to say no? That the impl will support whatever you put there as a subsecond and convert that to the precision of nanoseconds for the check against the log/event entries that are stored as seconds + nanoseconds. When they run the logs with -t they see seconds.nanoseconds.  When they cut and paste the timestamp from seconds.nanoseconds they would have to manually trunc some portion of the timestamp nanoseconds to get millis and it would work...  

I propose leaving it as it is but add a sentence explaining it.  Something like "Where the optional .nanoseconds field is a fraction of a second and is no more than nine digits long."